src/ngraph/runtime/allocator.cpp

```diff
@@ -51,6 +51,6 @@ class ngraph::runtime::DefaultAllocator : public ngraph::runtime::Allocator
 
 ngraph::runtime::Allocator* ngraph::runtime::get_default_allocator()
 {
-    static std::unique_ptr<DefaultAllocator> allocator(new DefaultAllocator());
-    return allocator.get();
+    static DefaultAllocator* allocator = new DefaultAllocator();
```

Comment:
I would approve of the code that @aprocter suggested above.

Reply:
(Comment for posterity: see the top-level PR thread. This is probably the right thing here.)

Reply:
Let's switch this to 0.24 as a hot fix and work out the ownership policy.